### ISSUE TITLE
accessibility for icons; replace i tags with em.

### ISSUE THREAD

**Issue by noelzubin** [ant-design/ant-design#19829]
- [x] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### Reproduction link
[https://ant.design/components/icon/](https://ant.design/components/icon/)

### Steps to reproduce
Non svg icons are rendered by default in a i tag.

### What is expected?
Icons should be rendered in an `em` tag instead for accessibility reasons.

### What is actually happening?
Icons are rendered in an i tag instead of em tag.

| Environment | Info |
|---|---|
| antd | 3.25.2 |
| React | * |
| System | * |
| Browser | * |

---
accessibility tools to refer to:   
https://www.deque.com/axe/  
https://wave.webaim.org/help  

https://stackoverflow.com/questions/40264695/font-awesome-accessibility-issue-use-em-tags-instead-of-i

**Comment by afc163** [comment]
> With the use of ligatures, using the i tag to display icons is semantic. In fact, Google suggests the use of the i tag with ligatures in it's Material icons guide.

https://stackoverflow.com/questions/11135261/should-i-use-i-tag-for-icons-instead-of-span#comment50502448_11135302

Also, we add `aria-label` for a11y support.

<img width="463" alt="image" src="https://user-images.githubusercontent.com/507615/69221885-4309f900-0bb3-11ea-8243-0f24aca8f315.png">
